internal/registry/registry.go

```diff
@@ -46,18 +46,15 @@ type Registry interface {
 	GetLayerByDigest(digest string, pullConfig *RepoPullConfig) (v1.Layer, error)
 }
 
-type registry struct{}
+type registry struct {
+}
```

Comment:
Nit, we can close the diff here.

Reply:
this change was done by fmt, so i would stick with it